.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+resolves #
+
+<!---
+  Include the number of the issue addressed by this PR above if applicable.
+  PRs for code changes without an associated issue *will not be merged*.
```

Comment:
Are we holding to this? I'm fine with it but we'll need to retrain all of the existing MetricFlow devs, since we didn't do this at all in the past.

Reply:
Personally I am a fan of this as it makes sure that there is always an issue record for changes. That said, if you think it will slow down velocity I'm happy to remove this section.

Reply:
Actually the other thing is that `changie` requires that an issue number be provided with the changelog entry so this might be required